python/ray/serve/handle.py

```diff
@@ -216,31 +240,35 @@ def deployment_name(self) -> str:
     def app_name(self) -> str:
         return self.deployment_id.app_name
 
+    @property
+    def is_initialized(self) -> bool:
+        return self._router is not None
+
+    def _init(self, **kwargs):
+        if self._router is not None:
+            raise RuntimeError("Handle has already been initialized.")
```

Comment:
let's add some more context here to help an uninformed user

Reply:
added!